parent/pom.xml

```diff
@@ -708,7 +708,6 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-deploy-plugin</artifactId>
                 <version>2.8.2</version>
-                <inherited>false</inherited>
```

Comment:
Maybe we could specify skip=true for the modules that we do not want to publish? Keep the default true. Otherwise we will eventually ship version with some of new modules not in Maven.

Reply:
The whole idea was in changing default policy to `skip`, rather than `include`.
So that we did not push something we should not (like `extdata` in this issue).
Otherwise most of the changes in this PR make no sense.

However, I think we need some place in documentation, where this should be described.
@dmagda, can you help us?

Reply:
I don't think this is about documentation - at least not the user documentation. Maybe just add it as a comment to the parent pom.xml.

Reply:
I ment "How to contribute" (or alike) doc in Apache Ignite Wiki.